docs/third-party-cdn.md

```diff
@@ -0,0 +1,41 @@
+# Third Party CDN Usage
+
+In some situations it may be required to use your own CDN to serve the Altis powered website or service. Below are some examples that may necessitate this:
+
+- Many web properties are served from a single domain name, whereby you need to control routing. A sub-section of the URL namespace may route to Altis, in which case it's not possible to route all traffic to Altis' CDN for a single domain name.
+- There may have compliance requirements or company-wide mandated use of a specific CDN vendor.
+
+In these cases, Altis Cloud can be configured to serve traffic to your CDN via an origin-pull setup. Altis does not support an origin-push CDN  setup. Reach out to Altis Cloud Support for assistance in third party CDN integration as it requires specific manual work be done.
+
+## CDN Configuration
+
+Altis Cloud will provide origin DNS names for each Cloud environment (`development`, `staging` and `production`) to set as the Origin-Pull upstream for you CDN. For example, this could be `development.example.altis.cloud`. The CDN should be configured with the following logic, in whatever format or system the CDN uses.
+
+### Cookies
+
+All cookies matching the pattern `wordpress_*`, `wp-*`, `wp_*`, `comment_*` and `hm_*` should be forwarded to the Altis Cloud origin.
+
+### URL Query Parameters
+
+All URL query parameters should be forwarded to the Altis Cloud hostname.
+
+### HTTP Headers
+
+The following HTTP headers should be forwarded: `Host`, `Origin`, `Authorization`.
```

Comment:
Accept too?

Reply:
Hmm we don't forward that header now. I added a note on calculating WebP support cased off Accept on the CDN edge node.